src/pretalx/schedule/exporters.py

```diff
@@ -160,6 +161,15 @@ def get_data(self, **kwargs):
                 "end": self.event.date_to.strftime("%Y-%m-%d"),
                 "daysCount": self.event.duration,
                 "timeslot_duration": "00:05",
+                "rooms": [
```

Comment:
Is this something that is done by frab and/or supported by dependent apps? I don't think there exists a schema for the frab JSON exports …

Reply:
Not yet, see https://github.com/frab/frab/issues/507 – I can also implement this in frab, if you prefer that :-)

Reply:
I plan to create a json-schema file for the schedule.json format, but have not done so – yet. And I am not aware of any person who has done that.